index.html

```diff
@@ -2089,8 +2089,7 @@ <h3 id="candidate-rec"><a id="last-call">6.4 Candidate Recommendation </a></h3>
           new Candidate Recommendation.</li>
       </ul>
       <p>The Director <em class="rfc2119">must</em> announce the publication of a Candidate Recommendation to other W3C groups
-        and to the public, and <em class="rfc2119">must</em> begin an Advisory Committee Review on the question of whether the
-        specification is appropriate to publish as a W3C Recommendation.</p>
+        and to the public.</p>
```

Comment:
You need to start the review somewhere. And this should be logged as a change.

SUbmitted a PR against the branch.